file_catalog/server.py

```diff
@@ -125,6 +125,10 @@ def create(config: Dict[str, Any],
         "config": config,
         "db": mongo,
         "debug": debug,
+        "route_stats": {
+            "window_time": config["ROUTESTATS_WINDOW_SIZE"],
```

Comment:
I'll lobby that you change the env var to `ROUTESTATS_WINDOW_TIME` because the RouteStats object has both a `window_size` and `window_time` field, and this is supplying a value for `window_time` while being named `ROUTESTATS_WINDOW_SIZE`, could be confusing.

I didn't realize at first that OVERLOAD_LIMIT was going to `timeout`; this one might be renamed `ROUTESTATS_TIMEOUT` instead, but this is just a suggestion.

Reply:
Sure, I can do those.